.gitlab-ci.yml

```diff
@@ -6,72 +6,82 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 ###############################################################################
 
+# DESCRIPTION:
 ###############################################################################
 # General GitLab pipelines configurations for supercomputers and Linux clusters
 # at Lawrence Livermore National Laboratory (LLNL).
-#
 # This entire pipeline is LLNL-specific
 #
-# Important note: This file is a template provided by
-# llnl/radiuss-shared-ci. Changes needed consists in setting variable values,
-# change the reference to the radiuss-shared-ci repo, opt-in and out optional
-# features. The project can then extend it with additional stages.
+# Important note: This file is a template provided by llnl/radiuss-shared-ci.
+# Remains to set variable values, change the reference to the radiuss-shared-ci
```

Comment:
`Remains to set` is a bit awkward, and I'm not sure how this should be rephrased.